packages/backend/src/server/api/RateLimiterService.ts

```diff
@@ -31,76 +31,54 @@ export class RateLimiterService {
 	}
 
 	@bindThis
-	public limit(limitation: IEndpointMeta['limit'] & { key: NonNullable<string> }, actor: string, factor = 1) {
-		{
-			if (this.disabled) {
-				return Promise.resolve();
-			}
-
-			// Short-term limit
-			const min = new Promise<void>((ok, reject) => {
-				const minIntervalLimiter = new Limiter({
-					id: `${actor}:${limitation.key}:min`,
-					duration: limitation.minInterval! * factor,
-					max: 1,
-					db: this.redisClient,
-				});
-
-				minIntervalLimiter.get((err, info) => {
-					if (err) {
-						return reject({ code: 'ERR', info });
-					}
+	private checkLimiter(options: Limiter.LimiterOption): Promise<Limiter.LimiterInfo> {
+		return new Promise<Limiter.LimiterInfo>((resolve, reject) => {
+			new Limiter(options).get((err, info) => {
+				if (err) {
+					return reject({ code: 'ERR', info });
+				}
```

Comment:
ここ

```suggestion
				if (err) {
					return reject(info);
				}
```

にしたほうがAPIの意図としては正しそうですが、できる限り元の実装から変えないようにしたためこのようにしてます

Reply:
f9dd3aca785dc9cec5c4f2a7abc746987b534e1a でそもままrejectするようにしました